src/providers/FiltersProvider.tsx

```diff
@@ -70,6 +70,7 @@ const imageFilterSchema = z.object({
   followed: z.boolean().optional(),
   tools: z.number().array().optional(),
   techniques: z.number().array().optional(),
+  baseModels: z.enum(constants.baseModels).or(z.enum(constants.baseModels).array()).optional(),
```

Comment:
Here we can use the helper method `commaDelimitedEnumArray`, this will make it so that users can pass a comma separated string in the same query, for example: `&baseModels=SD 1.5,SDXL,Pony`
```suggestion
  baseModels: commaDelimitedEnumArray(z.enum(constants.baseModels)).optional(),
```